src/index.tsx

```
@@ -24,121 +1,144 @@
-      authorization: 'Bearer ' + getItem('token') || '',
-      'Accept-Language': lng,
-    },
-  };
-});
+	const { getItem } = useLocalStorage();
+	const authLink = setContext((_, { headers }) => {
+		const lng = i18n.language;
+		return {
+			headers: {
+				...headers,
+				authorization: 'Bearer ' + getItem('token') || '',
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)